Sources/SwiftFormatPrettyPrint/PrettyPrint.swift

```diff
@@ -275,10 +278,18 @@ public class PrettyPrinter {
         // lines within it (unless they are themselves continuations within that particular
         // scope), so we need the continuation indentation to persist across all the lines in that
         // scope.
-        activeOpenBreaks.append(
-          ActiveOpenBreak(
-            lineNumber: currentLineNumber,
-            contributesContinuationIndent: currentLineIsContinuation))
+        var activeOpenBreak = ActiveOpenBreak(
+          kind: openKind,
+          lineNumber: currentLineNumber,
+          contributesContinuationIndent: currentLineIsContinuation)
+        activeOpenBreak.contributesBlockIndent = openKind == .block
+
+        // Check if this break will fire, and apply a continuation indent to its contents.
```

Comment:
Can this section (up until the `append` call) be moved above where the `ActiveOpenBreak` is created, so that instead of reassigning a value to `contributesContinuationIndent`, we start a temporary variable (perhaps also named `contributesContinuationIndent`) equal to `currentLineIsContinuation`, then update it based on this, and pass that into the `ActiveOpenBreak` initializer? That, along with the change above, would remove all the state changes in this section of the code, so it may be easier to reason about.